chapters/managementCompanies.md

```diff
@@ -0,0 +1,19 @@
+# Companies management
+## Show all companies
+The *Show Company* button in the menu bar allows you to give access to all companies recorded in MaaS. The list of companies will be shown in tabular form.
+![](../img/showCompanies.png)
+
+## Remove a company
+Clicking on a company name, you can see the button to remove it from MaaS.
```

Comment:
**By** clicking on a company name, you can see the _delete_ button that allows you to delete the company.